### PR TITLE
Removed duplicate DNPronounced state causing upload error

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###

None

### Change description ###

Removed duplicate DNPronounced state causing upload error

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 1]
```diff
@@ -1273,34 +1273,13 @@
...
     "UserRole": "caseworker-divorce-courtadmin",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "DNPronounced",
-    "UserRole": "caseworker-divorce-superuser",
-    "CRUD": "RU"
-  },
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseState.json]: Not 100% sure which permissions should stay, check the existing entry (some have only R)

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 2]
```diff
@@ -1273,34 +1273,13 @@
...
-    "UserRole": "caseworker-divorce-courtadmin-la",
-    "CRUD": "RU"
-  },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "DNPronounced",
-    "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "RU"
-  },
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseState.json]: Existing one has
```
  {
    "LiveFrom": "09/11/2018",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "DNPronounced",
    "UserRole": "caseworker-divorce-courtadmin_beta",
    "CRUD": "CRU"
  },
```

File: definitions/divorce/json/AuthorisationCaseState.json [hunk 3]
```diff
@@ -1273,34 +1273,13 @@
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
   },
-  {
-    "LiveFrom": "25/06/2019",
-    "CaseTypeID": "DIVORCE",
-    "CaseStateID": "DNPronounced",
-    "UserRole": "caseworker-divorce-courtadmin-la",
-    "CRUD": "RU"
-  },
```
Comment (hosainnet) [inline definitions/divorce/json/AuthorisationCaseState.json]: Existing one has just R
```
  {
    "LiveFrom": "23/05/2019",
    "CaseTypeID": "DIVORCE",
    "CaseStateID": "DNPronounced",
    "UserRole": "caseworker-divorce-courtadmin-la",
    "CRUD": "R"
  },
```

**Comment by hosainnet** [comment]
@tchow8 I've posted the existing auth fields as each has a different `CRUD` value - are you best to confirm the existing ones I left are accurate?

**Comment by tchow8** [comment]
Should be good :+1:

**Pull request merged**